.github/actions/auto-major-minor-release-v1/action.yml

```diff
@@ -0,0 +1,168 @@
+name: Automatically create patch release every 2 weeks
+description: Triggered on schedule by our integrations to patch release
+inputs:
+  axe-version:
+    required: true
+    type: string
+  release-command:
+    required: true
+    type: string
+  release-branch:
+    required: true
+    type: string
+  default-branch:
+    required: true
+    type: string
+  token:
+    required: true
+    type: string
+  project_token:
+    required: true
+    type: string
+  slack_webhook:
+    required: true
+    type: string
+runs:
+  using: "composite"
+  steps:
+    # Unlike patch release, it is expected that the integration script does work beforehand
+    # Specifically, updating axe-core and calculating major vs minor release is done before this
+    # gets called.
+
+    - name: Cancel if we already have a PR
+      uses: dequelabs/axe-api-team-public/.github/actions/cancel-if-release-pr-exists@main
+      with:
+        token: ${{ inputs.token }}
+
+    - name: Create axe-core update commit
+      shell: bash
+      run: |
+        git config user.name "API Team CI User"
+        git config user.email "aciattestteamci@deque.com"
+        git commit -am "feat: update axe-core to v$version"
+      env:
+        version: ${{ inputs.axe-version }}
+    - name: Run preparation script
+      shell: bash
+      run: ${{ inputs.release-command }}
```

Comment:
I don't know how I feel about allowing our script to execute an arbitrary bash command.

Reply:
This is the same thing we are doing in the auto-patch action.